crates/gpui/build.rs

```diff
@@ -324,6 +325,21 @@ mod windows {
         }
     }
 
+    fn find_windows_sdk_bin() -> String {
+        if let Ok(key) = RegKey::predef(HKEY_LOCAL_MACHINE)
+            .open_subkey("SOFTWARE\\WOW6432Node\\Microsoft\\Microsoft SDKs\\Windows\\v10.0")
+        {
+            let install_folder: String = key.get_value("InstallationFolder").unwrap_or_default(); // "C:\Program Files (x86)\Windows Kits\10\"
+            let product_version: String = key.get_value("ProductVersion").unwrap_or_default(); // "10.0.22621" NOTE: does not include trailing `.0`
+
+            if !install_folder.is_empty() && !product_version.is_empty() {
+                return format!("{}bin\\{}.0\\x64\\", install_folder, product_version);
```

Comment:
there's a very rare possibility of the version having the patch number in which case the .0 would theoretically break, e.g. 10.0.22621.1778, ideally a check would be added to ensure we don't create a non existing path